client/modules/core/main.js

```diff
@@ -186,21 +186,33 @@ export default {
     // called out a userId is validated.
     //
     function roleCheck() {
+      // TODO: There're some outdated comments in this function. Clean them out.
```

Comment:
For the outdated comments, you can make that change in this PR (i.e remove them). That will start a discussion here about if it needs to change or not

Reply:
Roger that. I've done some, but left with a few.

Reply:
But what should I do to [this commented code](https://github.com/reactioncommerce/reaction/blob/18ac96a140173753d12fbaf7b9d911a86503d31b/client/modules/core/main.js#L216)? I wasn't the one who wrote it, so I don't know it still is/will be useful.

Reply:
Looking at that piece of commented code, it's checking to see if a user has `admin` role for shops then based on that, it decides hasPermissions access (i.e return `true` or `false`). So, if a user has `admin` role, they get a pass. 
That can lead us to the same situation we're in with the `owner` role being concatenated in (i.e what you're fixing). I think that's why it was commented out. So, it's fine to remove it, since there are other ways to perform admin check.

Reply:
Done.

Reply:
Cool. Do you want me to do another review? or still working on other parts?

Reply:
Yes, you can perform another review. I've pushed changes to address the remaining part I was working on (see comments on main thread).